### ISSUE TITLE
Sidebar Widgets: MailChimp Subscriber Pop-Up Disables Ability to Add Woo Customers

### ISSUE THREAD

**Issue by tmmbecker** [Automattic/jetpack#7523]
#### Steps to reproduce the issue

1. Add the MailChimp Subscriber Pop-up widget - with a connection to a MailChimp list - to a site that uses WooCommerce
2. In WooCommerce -> Settings -> Account, make sure that "Enable registration on the "My Account" page" is enabled.
3. Go to the "My Account" page for the site.
4. Try to register as a new customer. 

#### What I expected

We should be able to add customers without any interference from Jetpack. 

#### What happened instead

The Register button disabled as soon as I started typing a password. 

![screen shot 2017-07-25 at 10 45 31](https://user-images.githubusercontent.com/22805865/28578053-8681997e-7126-11e7-9a97-d2fa62abb3e2.png)

I've left this up on my site if you'd like to see it in action. It's a live site, but not in use at the moment. 

https://deerdash.com/my-account/

Internal reference: 606355-zen

cc @pmciano

**Comment by mbhthompson** [comment]
Update: this bug is happening when any Jetpack module is activated (not just the Extra Sidebar Widgets module).

**Comment by eliorivero** [comment]
This is not a Jetpack issue. To start with, I deactivated Jetpack and tested. Same issue happened.

I checked it and found that it's related to pass length. Looked up the scripts loaded in the page and saw there's the password strength meter from core and one that uses it in WooCommerce. The relevant code is
https://github.com/woocommerce/woocommerce/blob/master/assets/js/frontend/password-strength-meter.js#L32
